src/bootstrap/config.rs

```diff
@@ -654,19 +655,21 @@ impl Config {
                 check_ci_llvm!(llvm.release_debuginfo);
                 check_ci_llvm!(llvm.link_shared);
                 check_ci_llvm!(llvm.static_libstdcpp);
-                check_ci_llvm!(llvm.targets);
-                check_ci_llvm!(llvm.experimental_targets);
                 check_ci_llvm!(llvm.link_jobs);
                 check_ci_llvm!(llvm.link_shared);
-                check_ci_llvm!(llvm.clang_cl);
-                check_ci_llvm!(llvm.version_suffix);
-                check_ci_llvm!(llvm.cflags);
-                check_ci_llvm!(llvm.cxxflags);
-                check_ci_llvm!(llvm.ldflags);
                 check_ci_llvm!(llvm.use_libcxx);
-                check_ci_llvm!(llvm.use_linker);
                 check_ci_llvm!(llvm.allow_old_toolchain);
 
+                // N.B. these use `config` because they were moved above
```

Comment:
I don't think we should complicate the code for the cost of a few clones -- remember that bootstrap isn't even compiled in release mode, performance *does not matter*.

Reply:
Ok, I reverted the changes.